tools/IoTEdgeDevOps/VstsPipelineSync/db/dbo.VstsBug.sql

```diff
@@ -6,14 +6,16 @@ SET QUOTED_IDENTIFIER ON
 GO
 
 CREATE TABLE [dbo].[VstsBug](
-	[QueryName] [varchar](20) NOT NULL,
+	[Title] [varchar](200) NOT NULL,
+	[AreaPath] [varchar](200) NOT NULL,
+	[Priority] [varchar](20) NOT NULL,
+	[InProgress] [bit] NOT NULL,
 	[BugCount] [int] NOT NULL,
 	[InsertedAt] [datetime2](7) NOT NULL,
 	[UpdatedAt] [datetime2](7) NOT NULL,
  CONSTRAINT [PK_QueryName] PRIMARY KEY CLUSTERED 
 (
-	[QueryName] ASC
+	[Title] ASC
```

Comment:
Suggestion: add a datetime field as part of primary key; therefore this table will keep number of bugs for every day.  This may help to answer more question later.  If this is too big to change, forget it.

Reply:
I feel it's best to wait until we have a plan for what to do with this historical data. Maybe display a graph of bugs over time? I'd have to think of how best to display this because lots of areas/categories.